glusterd2/volume/store-utils.go

```diff
@@ -218,3 +218,14 @@ func Exists(name string) bool {
 
 	return resp.Count == 1
 }
+
+// CheckBrickExistence checks if a brick is part of a particular volume
```

Comment:
can you update the comment to something like "CheckBrickExistence checks if a brick is part of a host in a volume"
Want this to be mentioned as it checks against the hostname as well.